cli_reference/index.adoc

```diff
@@ -0,0 +1,38 @@
+[id="cli-tools-overview"]
+= {product-title} CLI tools overview
+include::modules/common-attributes.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user performs different tasks while working on {product-title} which includes a range of operations such as the following:
```

Comment:
Is the use of 'user' instead of 'you' a deliberate one? As per ISG ('Guidance for all tones' section): Wherever possible, use the second person ("you"). 
I am not sure I understand the difference between 'different tasks' and 'a range of operations'. The list below could fall in the category of tasks as well as operations.
How about: A user performs a range of operations while working on {product-title} such as the following: